enterprise/frontend/src/embedding-sdk/hooks/private/use-load-question.ts

```diff
@@ -3,11 +3,11 @@ import { useAsyncFn, useUnmount } from "react-use";
 
 import type { ParameterValues } from "embedding-sdk/components/private/InteractiveQuestion/context";
 import {
-  runQuestionOnLoadSdk,
+  loadQuestionSdk,
```

Comment:
this file is for interactive question components, not static question component.